funcs_scripts.php

```diff
@@ -133,11 +184,38 @@ function info($message)
 
 /**
  * Output a warning message to the console
- * 
+ *
  * Example usage:
  * warning('This is something you might want to pay attention to')
  */
 function warning($message)
 {
     io()->warning($message);
 }
+
+/**
+ * Converts a cron expression to a human readable string
+ * Says UTC because that's what GitHub Actions uses
+ *
+ * Example usage:
+ * human_cron('5 4 * * 0')
+ * => 'At 4:05 AM UTC, only on Sunday'
+ */
+function human_cron(string $cron): string
+{
+    $str = (new ExpressionDescriptor($cron))->getDescription();
+    $str = preg_replace('#0([1-9]):#', '$1:', $str);
+    $str = preg_replace('# (AM|PM),#', ' $1 UTC,', $str);
+    return $str;
+}
+
+/**
+ * Creates a predicatable random int between 0 and $max based on the module name to be used with the % mod operator
```

Comment:
```suggestion
 * Creates a predicatable random int between 0 and $max based on the module name to be used with the % mod operator.
 * Note that this will return the exact same value every time it is called for a given module.
```
Just to make this abundantly clear.

Reply:
Updated